apps/docs/docs/learn/02-thinking-in-stylex.mdx

```diff
@@ -249,15 +249,15 @@ type Props = {
 ```
 
 Styles being typed enables extremely sophisticated rules about the ways in which
-a component styles can be customized with **zero-runtime cost**.
+a component's styles can be customized with **zero-runtime cost**.
 
 ### Shareable constants
 
-CSS class names, CSS variables and other CSS identifiers are defined in a global
-namespace. The obvious and common API to bring them into JavaScript is as
+CSS class names, CSS variables, and other CSS identifiers are defined in a global
+namespace. The obvious and common API to bring them into JavaScript are as
```

Comment:
The "is" is referring to "the API", so it should be an "is" and not "are".

Reply:
I just set it to the sentence @necolas provided, we can revert it but it was a little difficult to read